setup.py

```diff
@@ -0,0 +1,114 @@
+    long_description=_README,
+    classifiers=[
+        "Development Status :: 4 - Beta",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Education",
+        "Intended Audience :: Financial and Insurance Industry",
+        "Intended Audience :: Healthcare Industry",
+        "Intended Audience :: Information Technology",
+        "Intended Audience :: Manufacturing",
+        "Intended Audience :: Science/Research",
+        "Intended Audience :: System Administrators",
+        "Intended Audience :: Telecommunications Industry",
+        'License :: OSI Approved :: Apache Software License',
+        "Natural Language :: English",
+        "Operating System :: OS Independent",
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        "Topic :: Adaptive Technologies",
+        "Topic :: Office/Business",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Scientific/Engineering :: Artificial Intelligence",
+        "Topic :: Scientific/Engineering :: Artificial Life",
+        "Topic :: Scientific/Engineering :: Bio-Informatics",
+        "Topic :: Scientific/Engineering :: Image Recognition",
+        "Topic :: Scientific/Engineering :: Information Analysis",
+        "Topic :: Scientific/Engineering :: Interface Engine/Protocol Translator",
+        "Topic :: Scientific/Engineering :: Mathematics",
+        "Topic :: Scientific/Engineering :: Medical Science Apps.",
+        "Topic :: Scientific/Engineering :: Physics",
+        "Topic :: Software Development",
+        "Topic :: Software Development :: Assemblers",
+        "Topic :: Software Development :: Build Tools",
+        "Topic :: Software Development :: Libraries",
+        "Topic :: Software Development :: Libraries :: Python Modules",
+        # Topic :: Software Development :: Object Brokering,
+        "Topic :: Software Development :: Pre-processors",
+        "Topic :: Software Development :: Quality Assurance",
+        "Topic :: Software Development :: Testing",
+        "Topic :: System",
+        # Topic :: System :: Clustering,
+        # Topic :: System :: Distributed Computing,
+        # Topic :: System :: Networking,
+        # Topic :: System :: Systems Administration,
+        "Topic :: Text Processing",
+        "Topic :: Text Processing :: Filters",
+        "Topic :: Text Processing :: Linguistic",
+        "Topic :: Utilities",
+        "Typing :: Typed"
+    ],
+    url='https://github.com/Neuraxio/Neuraxle',
+    download_url='https://github.com/Neuraxio/Neuraxle/tarball/{}'.format(
+        _VERSION),
+    author='Neuraxio Inc.',
+    author_email='guillaume.chevalier@neuraxio.com',
+    packages=find_packages(include=['neuraxle-tensorflow*']),
+    test_suite="testing",
+    setup_requires=["pytest-runner"],
+    install_requires=[
+        'neuraxle>=0.2.1'
+    ],
+    extras_requires={
```

Comment:
haven't found any other way to specify tensorflow in the requirements. I think it's okey like this. what do you think ? See here : https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies @guillaume-chevalier

Reply:
I would not add TensorFlow in the requirements. I would let the user install it manually to the version he wants, and our lib should be compatible with the most recent versions of TensorFlow (e.g.: 1.14, 2.0).

Reply:
In other words, I would remove the `extra_requires` for now and just let users install the version of TF they want. If they don't install tf manually, it will just crash when trying to import tf.